timeseries/src/main/java/com/hurence/logisland/timeseries/sax/SaxAnalyzer.java

```diff
@@ -0,0 +1,75 @@
+package com.hurence.logisland.timeseries.sax;
+
+import com.sun.javafx.collections.MappingChange;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.*;
+
+
+public class SaxAnalyzer {
+
+    public static Map<Integer, Integer> countOccurrence(int[] numbersToProcess) {
+        int[] possibleNumbers = new int[30];
+        Map<Integer, Integer> result = new HashMap<Integer, Integer>();
+
+        for (int i = 0; i < numbersToProcess.length; ++i) {
+            possibleNumbers[numbersToProcess[i]] = possibleNumbers[numbersToProcess[i]] + 1;
+            result.put(numbersToProcess[i], possibleNumbers[numbersToProcess[i]]);
+        }
+
+        return result;
+    }
+    public static List<Integer> saxThreshold(String str){
+        double l = str.length();
```

Comment:
What is the purpose of this line since you use str.length() everywhere after... I suppose you can remove the statement

Reply:
I used it as a double so I can get float/double in this map:
Map<Integer, Double> buffCountNorm = buffCount.entrySet().stream().collect(toMap(e -> e.getKey(),e -> e.getValue()/(l-1)))

I was forced to convert map values into double or the string.length().

Reply:
good point ... okay then forget about my remark

Reply:
then you could have casted your str.length() to a double ;)
so ((Double)l - 1)

Reply:
sorry ((Double) str.length()) - 1)

Reply:
Ok I will cast str.length()